src/index.js

```diff
@@ -1 +1,7 @@
-module.exports = 'Placeholder'
+const { Client: MSTeamsClient } = require('./models/client')
+const { handler: eventMessageHandler } = require('./listeners/event-message')
+
+module.exports = {
+  MSTeamsClient,
+  eventMessageHandler
```

Comment:
`eventMessageHandler` seems like a parser for the poller, which I think should be enabled by default. I think it'd be better if you import that handler into the poller and call it there. If that works, remove the export from here.

Reply:
If I'm understanding you correctly, we would just call eventMessageHandler from the poller whenever an event type 'EventMessage' is received instead of from Lounge Lizard? I just did this and it seems to work fine.